css/modals/speaker_stats/_speaker_stats.scss

```diff
@@ -4,6 +4,34 @@
     width: 100%;
     font-weight: 500;
 
+    .speaker-stats-header {
```

Comment:
Could we please use the new design tokens as well as the material-ui styling instead of scss here? (See: https://github.com/jitsi/jitsi-meet/blob/master/react/features/participants-pane/components/LobbyParticipantList.js)

Reply:
Yes, it will be better, I will change it, thank you for your remark.

Reply:
I has been changed